searx/answerers/__init__.py

```diff
@@ -32,7 +32,7 @@ def ask(query):
     results = []
     query_parts = list(filter(None, query.query.split()))
 
-    if query_parts[0] not in answerers_by_keywords:
+    if len(query_parts) == 0 or query_parts[0] not in answerers_by_keywords:
```

Comment:
I would use : 

    if not query_parts or ...

Reply:
Done